examples/experiments/grounded_program_synthesis/train_trlx.py

```diff
@@ -50,6 +49,25 @@ def reward_fn(samples):
     return reward_list
 
 
+default_config = yaml.safe_load(open("config/trlx_ppo_config.yml"))
```

Comment:
wandb for this? (I know it's by @reshinthadithyan )

Reply:
I realize now you're hinting at #74

Reply:
Should I integrate wandb sweeps here?

Reply:
this commit should have integrated it. btw can you make your model public on hf? 🙂